installvm.py

```diff
@@ -321,21 +321,24 @@ def createKickstart(self):
                 host_disk += '/dev/disk/by-id/' + disk+','
             vmParser.args.host_disk = host_disk.rstrip(',')
 
-        if '8' not in version:
+        if version.startswith('8') or  version.startswith('9'):
             lstr = "telnet\njava\n%end"
```

Comment:
this line should go to else `lstr = "telnet\njava\n%end"`

Reply:
thanks for review , yes will adopt